plugins/device-injector/README.md

```diff
@@ -5,14 +5,33 @@ containers using pod annotations.
 
 ### Device Annotations
 
-Devices are annotated using the `devices.nri.io` annotation key prefix.
-The key `devices.nri.io/container.$CONTAINER_NAME` annotates devices to
-be injected into `$CONTAINER_NAME`. The keys `devices.nri.io` and
-`devices.nri.io/pod` annotate devices to be injected into containers
-without any other, container-specific device annotations. Only one of
-these latter two annotations will be ever taken into account. If both are
-present, `devices.nri.io/pod` is used and `devices.nri.io` is silently
-ignored, otherwise `devices.nri.io`, in the absence of additional suffix text,  is processed as shorthand for the `devices.nri.io/pod` annotation. The order of precedence is `devices.nri.io/container.$CONTAINER_NAME` is used, unless not present, then `devices.nri.io/pod` followed by the `devices.nri.io` shorthand annotation.
+Devices are annotated using the `devices.noderesource.dev` annotation key
+prefix.  The key `devices.noderesource.dev/container.$CONTAINER_NAME` annotates
+devices to  be injected into `$CONTAINER_NAME`. The keys
+`devices.noderesource.dev` and `devices.noderesource.dev/pod` annotate devices
+to be injected into containers  without any other, container-specific device
+annotations. Only one of  these latter two annotations will be ever taken into
+account. If both are  present, `devices.noderesource.dev/pod` is used and
+`devices.noderesource.dev` is silently ignored, otherwise
+`devices.noderesource.dev`, in the absence of additional suffix text, is
+processed as shorthand for the `devices.noderesource.dev/pod` annotation.
+
+For compatibility with older versions of this plugin, the prefix
+`devices.nri.io` is also supported and follows identical rules.  When
+equivalently-specific annotations are present with both the
+`devices.noderesource.dev` prefix and the `devices.nri.io` prefix, the
+annotation with the `devices.noderesource.dev` prefix is used and
+`devices.nri.io` is silently ignored.  In all cases, the most-specific
+annotation is preferred regardless of prefix.
+
```

Comment:
I see a few spots where AFAICT a single space have been replaced with two spaces. Is this intentional ?

Reply:
It's my habit to double space after periods.  Oops.

Reply:
Fixed